Components/ArticleComments.jsx

```diff
@@ -0,0 +1,12 @@
+export default function ArticleComments ({currentComments}) {
+    return (
+        <>
+             {currentComments.map(({body, author, votes, comment_id}) => {
+                    return <li key={comment_id}>
+                        <p>{body}</p>
+                        <p>Submitted by {author}, votes: {votes}</p>
+                    </li>
+                })}
+        </>
+    )
```

Comment:
Please also add a little message for the user to let them know the article has no comments if this is the case 🙂